frontend/wallet/src/main/DaemonProcess.re

```diff
@@ -1,40 +1,74 @@
 open Bindings;
 
-let start = port => {
-  print_endline("Starting graphql-faker");
-  let p =
-    ChildProcess.spawn(
-      "node",
-      [|
-        Filename.concat(
-          ProjectRoot.resource,
-          "node_modules/graphql-faker/dist/index.js",
-        ),
-        "--port",
-        string_of_int(port),
-        "--",
-        "schema.graphql",
-      |],
+module Command = {
+  type t = {
+    executable: string,
+    args: array(string),
+  };
+};
```

Comment:
We probably don't need this abstraction considering we will never run both of these processes in the real world (and won't actually run them both now, either)